node/cli/src/cli.rs

```diff
@@ -100,6 +100,11 @@ pub struct Cli {
 	#[cfg(feature = "frequency-no-relay")]
 	#[clap(long, help = "The interval in seconds", default_value = "12", value_name = "SECONDS")]
 	pub sealing_interval: NonZeroU16,
+
+	/// Whether to create empty blocks in manual and interval sealing modes.
+	#[cfg(feature = "frequency-no-relay")]
+	#[clap(long, help = "Create empty blocks in interval sealing modes", default_value = "false")]
```

Comment:
So we want to have the default true or false? I was imagining true and I guess that would mean inverting the flag name. Thoughts?

Reply:
No empty blocks seems more useful. Hence, the default of false. What would be the rationale for having empty blocks?

Reply:
It was decided to keep this default.